### ISSUE TITLE
[RC feedback] Global Search doesn't show results - Class 'AllAssets' not found

### ISSUE THREAD

**Issue by viktor-sc** [glpi-project/glpi#7514]
**Describe the bug**

Global Search doesn't show any results and there are errors in the browser as well as in the logs. Looks like it's related to #7286 #7294 but I use the version 9.5 RC2 where it should be fixed. I do see the changes of the mentioned bugfixes in the search.class.php.
I have the 'Objects management' plugin installed but the error appears also if it's deactivated.
Also, I changed the columns prior to the update from 9.4.6 to 9.5 RC2 so that it should show: Name, Status, Comments, Network - IP, Type of entry. In 9.4.6 it worked fine.

**To reproduce**

1. Open Assets > Global
2. The page doesn't load.

**Logs**

php-errors.log shows:
```
[2020-07-01 11:24:19] glpiphplog.CRITICAL:   *** Uncaught Exception Error: Class 'AllAssets' not found in /volume1/web/glpitest/inc/search.class.php at line 561
  Backtrace :
  inc/search.class.php:97                            Search::constructSQL()
  inc/search.class.php:80                            Search::showList()
  front/allassets.php:45                             Search::show()
 
```

**Screenshots**

![image](https://user-images.githubusercontent.com/21951987/86229429-fd250280-bb8f-11ea-8562-c7d3c0cb7c24.png)

**Comment by trasher** [comment]
#7515 should solve the problem, could you please test and confirm? Even if this fix is not yet validated, it would help to know if that fix your case aswell. Thank you!

**Comment by viktor-sc** [comment]
Hi,
after changing the lines in the search.class.php according to the fix you mentioned #7515 the global search openes successfully and shows a result.

**Comment by trasher** [comment]
Ok, thank you for the feedback. I'm working to find the better solution, I'll keep you in touch.
